### ISSUE TITLE
selectedIndex on options with optgroup

### ISSUE THREAD

**Issue by dmathieu** [smparkes/env-js#8]
Hello,

I'm testing some events on a select box.

So I define the select's value and then I trigger the event.

```
js("jQuery('#select')[0].selectedIndex = 1;")
js("jQuery('#select').change();")
```

But this select box now has some optgroups which it didn't have before.
And with these optgroups, envjs can't define the value (when it works in firebug).

I get the following error :

```
Johnson::Error in 'MainController should get the multi properties edit page'
```

this.parentNode.options is undefined
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:6779 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:6815 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7108 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7047 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7076 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:6775 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:6815 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7108 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7047 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7076 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:6786 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:6815 [JavaScript]
    /usr/lib64/ruby/gems/1.8/gems/envjs-0.2.0/lib/envjs/static.js:7108 [JavaScript]
    inline:1 [JavaScript]

**Comment by smparkes** [comment]
Thanks for the report. Some of our form objects are incomplete. Can you post a copy of the form, too? That way I can create a test and make sure it gets (and stays) fixed.

**Comment by dmathieu** [comment]
Thanks :)
The form has the following :

```
<select id="select">
    <option name="">First Option</option>
    <optgroup label="Text">
        <option name="edit">Edit</option>
        <option name="delete">Delete</option>
    </optgroup>
    <optgroup label="Some Other Test">
        <option name="download">Download</option>
    </optgroup>
</select>
```

**Comment by smparkes** [comment]
That'll help a lot. I'm really slammed, maybe for a week or so, but ...

**Comment by smparkes** [comment]
Okay. Sorry for the delay. Confirmed the issue. More soon.

**Comment by smparkes** [comment]
Fixed in 0.3.2.
